tests/ProxyManagerTest/GeneratorStrategy/FileWriterGeneratorStrategyTest.php

```diff
@@ -47,7 +47,7 @@ protected function setUp() : void
         parent::setUp();
 
         $this->tempDir              = tempnam(sys_get_temp_dir(), 'FileWriterGeneratorStrategyTest');
-        $this->originalErrorHandler = static function () : bool {
+        $this->originalErrorHandler = static function () : void {
```

Comment:
This is supposed to stay `: bool`

Reply:
I've reverted that, but just for my understanding: This is needed because you defined the return type of the second argument using `@psalm-param` here: https://github.com/Ocramius/ProxyManager/blob/dfc740709aab9a334b75ba09ef948830c3be2aa9/src/ProxyManager/Factory/LazyLoadingGhostFactory.php#L87

Is that right? The static function doesn't really return anything or is there some magic I can't see?

Reply:
Correct: the function doesn't return anything, but a `callable() : bool` is expected by callees